custom-elements/upgrading/Node-cloneNode.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: Upgrading</title>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<meta name="assert" content="Node.prototype.cloneNode should upgrade a custom element">
+<link rel="help" href="https://html.spec.whatwg.org/#upgrades">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+
+function withNewDocumentWithABrowsingContext(callback) {
```

Comment:
Unfortunately, this variant removes the iframe it added before ending the test so the semantics is quite different. I moved this to custom-elements-helpers.js so that we can reuse it elsewhere.